bundle/camunda-saas-bundle/src/test/java/io/camunda/connector/runtime/saas/TestSpringContextStartup.java

```diff
@@ -48,17 +46,4 @@ public void contextLoaded() {
     // when the Spring context is initialized (e.g.
     // https://github.com/camunda/team-connectors/issues/251)
   }
-
-  @Test
-  public void operatePropertiesAreSet() {
```

Comment:
Should we replace these with a new test?

Perhaps one that checks the JWT credential is configured correctly?

Reply:
Done!